server.js

```diff
@@ -415,8 +419,25 @@ function prStatus (pr, delta, cb) {
                                         pr.contribStatus[username] = "ok";
                                         return cb(null, "ok");
                                     } else {
-                                        pr.contribStatus[username] = "not in group";
-                                        return cb(null, "not in group");
+                                        log.info("Looking up for non-participant licensing contribution");
```

Comment:
Maybe only check nplc for WGs (since they can't exist for CGs)

Reply:
Done in https://github.com/w3c/ash-nazg/commit/c5291ccd6ce483b2e738e7c2d15962dae6776e19